kratos/processes/from_json_check_result_process.h

```diff
@@ -276,26 +277,24 @@ class KRATOS_API(KRATOS_CORE) FromJSONCheckResultProcess
         for (auto& p_var_double : mpNodalVariableDoubleList) {
             const auto& r_var_database = r_node_database.GetVariableData(*p_var_double);
 
-            #pragma omp parallel for reduction(+:check_counter)
-            for (int i = 0; i < static_cast<int>(r_nodes_array.size()); ++i) {
-                auto it_node = it_node_begin + i;
-
+            check_counter = IndexPartition<std::size_t>(r_nodes_array.size()).for_each<SumReduction<IndexType>>([&] (std::size_t Index){
```

Comment:
@ashishdarekar I think it would be better to only capture the things you need and not everything, since this might hide errors

Reply:
Okay. I will do this in the next PR.